common/src/main/java/com/wynntils/services/mapdata/attributes/impl/AbstractMapAttributes.java

```diff
@@ -64,7 +64,7 @@ public Optional<MapDecoration> getIconDecoration() {
     }
 
     @Override
-    public Optional<MarkerOptions> getMarkerOptions() {
+    public Optional<MapMarkerOptions> getMarkerOptions() {
```

Comment:
MapMarkerOptions? Technically it's also in world markers, in fact, it mostly is.

Reply:
I see your point. And sure, we can change it back if you really want. My thinking was to try to get a common prefix for all "map data". And even though we display it in the "world", it is still map data. I mean, you could technically tint the world the same color as the color of the territory you are currently in. So to me "map data" means like "geography-based information"; but it does not strictly have to be displayed on a map. It is also used in the `/locate` command, etc.

So maybe we should call it `GeoData` instead? 🤔

Reply:
I'm not sure if that's better 😅

Reply:
Or we can use the precedence from GIS and call it ["geospatial data"](https://en.wikipedia.org/wiki/Geographic_data_and_information). ;-)

Or we stick to the "Map" but are clear (perhaps by editing the glossary wiki) that this term refers to geospatial data, which are not strictly necessarily displayed on a map (even if that is the primary use case).